content/guides/testing-strategies/working-with-graphql.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Working with GraphQL
+e2eSpecific: true
```

Comment:
It looks like this page isn't e2e specific, it's just this specific example needs to have a tabbed example https://docs.cypress.io/guides/testing-strategies/working-with-graphql#Modifying-a-Query-or-Mutation-Response

Please revert this change and I'll move the line in the spreadsheet to UNIFY-506.